hrjobcontract/js/src/job-contract/controllers/contract.js

```diff
@@ -82,10 +82,12 @@ define([
 
         if (isCurrentContract) {
           $scope.$parent.contract.is_current = '1';
+          $scope.$parent.contractCurrent = _.uniq($scope.$parent.contractCurrent);
```

Comment:
As it may hold `0` or more current contracts, changing it requires to change the name in other places in $scope. This is correct as `$scope.$parent.currentContract`.